application/utils.py

```diff
@@ -0,0 +1,17 @@
+import os
+import httpx
+from dotenv import load_dotenv
+
+
+load_dotenv()
+
+API_KEY = os.environ.get("API_KEY")
+URL = "https://api.weatherapi.com/v1/current.json?"
+
+
+async def get_temperature_from_weatherapi(city_name: str) -> int:
+    async with httpx.AsyncClient() as client:
+        response = await client.get(URL + f"q={city_name}&key={API_KEY}")
+        response.raise_for_status()  # Raises an error for non-200 responses
+        data = response.json()
```

Comment:
The `response.json()` method can raise an error if the response body is not valid JSON. Consider adding error handling to manage such cases gracefully.